pkg/api/types.go

```diff
@@ -1092,6 +1092,7 @@ const (
 	ClusterProfileAzure4                ClusterProfile = "azure4"
 	ClusterProfileAzureArc              ClusterProfile = "azure-arc"
 	ClusterProfileAzureStack            ClusterProfile = "azurestack"
+	ClusterProfileAWSOSDMSP             ClusterProfile = "awsosdmsp"
```

Comment:
Sort this together with other AWS linkes, please. Also, please make this respect the convention and call it `aws-<name>` instead, like all other ones are called